Sources/SmithyEventStreams/DefaultMessageEncoder.swift

```diff
@@ -0,0 +1,22 @@
+//
+// Copyright Amazon.com Inc. or its affiliates.
+// All Rights Reserved.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+import struct SmithyEventStreamsAPI.Message
+import protocol SmithyEventStreamsAPI.MessageEncoder
+import struct Foundation.Data
+
+    /// Encodes a `Message` into a `Data` object
+    /// to be sent over the wire.
+    public struct DefaultMessageEncoder: MessageEncoder {
+        public init() {}
+
+        /// Encodes a `Message` into a `Data` object
+        public func encode(message: Message) throws -> Data {
+            let crtMessage = message.toCRTMessage()
+            return try crtMessage.getEncoded()
+        }
+    }
```

Comment:
reminder: Extra indent for entire struct here as well, in addition to commented one in `Sources/SmithyEventStreams/DefaultMessageEncoderStream.swift`

Reply:
Yeah, those will all be cleaned up later.

There will likely be some time spent on cleanup across the board after all the modules have been completed.